packages/maskbook/src/plugins/Airdrop/apis/index.ts

```diff
@@ -1,3 +1,21 @@
-export function fetchNameList() {
-    return []
+export interface AirdropPacket {
+    address: string
+    index: string
+    amount: string
+    proof: string[]
+}
+
+export async function getMaskAirdropPacket(address: string) {
+    try {
+        const response = await fetch('https://service.r2d2.to/airdrop-lookup', {
```

Comment:
plz check response status code is non-ok status

see https://github.com/DimensionDev/airdrop-lookup-lambda/blob/master/lambda.ts

Reply:
For now, it's fine. So I'm letting it go. Or could you please do me a favor?